src/Sulu/Bundle/AdminBundle/DependencyInjection/Configuration.php

```diff
@@ -47,7 +47,7 @@ public function getConfigTreeBuilder()
             ->children()
                 ->arrayNode('resources')
                     ->useAttributeAsKey('resourceKey')
-                    ->arrayPrototype()
+                    ->prototype('array')
```

Comment:
arrayPrototype only exists since 3.3 prototype('array') need to be used instead: http://symfony.com/doc/3.3/components/config/definition.html#array-nodes